.github/scripts/update_version.py

```
@@ -0,0 +1,45 @@
+
+    const_path = PROJECT_ROOT / "custom_components/variable/const.py"
+    const = const_path.read_text()
+    const, replacements = re.subn(
+        r'^VERSION = ".*"$',
+        f'VERSION = "{tag}"',
+        const,
+        count=1,
+        flags=re.MULTILINE,
+    )
+    if replacements != 1:
+        raise RuntimeError("Unable to update VERSION in const.py")
```

Ruff (0.16.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)